tests/integration/tests/test_util/util.py

```diff
@@ -261,3 +262,24 @@ def get_default_ip(instance: harness.Instance):
         ["ip", "-o", "-4", "route", "show", "to", "default"], capture_output=True
     )
     return p.stdout.decode().split(" ")[8]
+
+
+def get_global_unicast_ipv6(instance: harness.Instance, interface="eth0") -> str:
```

Comment:
I see a potential for flakiness here, interface eth0 is a legacy name since [Predictable Network Interface Names](https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/) are being used that are more precise.

Reply:
Not sure if I understand? What is the flake in that case?
We explicitly ask for the IPv6 address of the `eth0` interface and this interface is explicitly set by LXD.

Reply:
Okay, then this doesn't apply. I was concerned that we might not always have eth0 as the interface name.